internal/helpers/prometheus_helper.go

```diff
@@ -54,15 +69,20 @@ func CreatePrometheusRule(slo *openslov1.SLO, sli *openslov1.SLI) (*monitoringv1
 		OwnerReferences: ownerRef,
 	}
 
-	finalMonitoringRules := make([]monitoringv1.Rule, len(monitoringRules))
-	for i, localRule := range monitoringRules {
-		finalMonitoringRules[i] = monitoringv1.Rule(localRule)
+	finalSimpleMonitoringRules := make([]monitoringv1.Rule, len(simpleMonitoringRules))
+	for i, localRule := range simpleMonitoringRules {
+		finalSimpleMonitoringRules[i] = monitoringv1.Rule(*localRule)
+	}
+
+	finalTimeWindowedMonitoringRules := make([]monitoringv1.Rule, len(timeWindowedMonitoringRules))
+	for i, localRule := range timeWindowedMonitoringRules {
+		finalTimeWindowedMonitoringRules[i] = monitoringv1.Rule(*localRule)
 	}
 
 	ruleGroup := []monitoringv1.RuleGroup{
 		{
 			Name:  slo.Name,
-			Rules: finalMonitoringRules,
+			Rules: slices.Concat(finalSimpleMonitoringRules, finalTimeWindowedMonitoringRules),
```

Comment:
I will need some enlightenment around this 🤔 If I'm not mistaken, you can use `slices.Concat()` method from Go 1.22, but we are running osko on version 1.21. You should also bump the Go version in `go.mod`, if you want to go this way.

Reply:
Let's merge in other PR